### DISCUSSION TITLE
RAG Document upload not handling 413 error with proxy (size too big)

### DISCUSSION THREAD

**Discussion by MHugonKaliop** [open-webui/open-webui#3040] [General]
# Bug Report

## Description

**Bug Summary:**
I use an Nginx proxy to handle SSL connection to openUI.
If I try to upload a document exceeding the max value configured in Nginx, on WebUI interface, nothing shows that an error occured. As an enduser, I'll wait undefinitely, I'm still seeing the "work in progress" animation, even if the web console shows that the server answered with an HTTP 413 error

**Steps to Reproduce:**
Use a Nginx proxy between the user browser and OpenWebUI application. 
Try to upload a file exceeding Nginx configuration

**Expected Behavior:**
An error should be seen on the interface, saying that the file is too large

**Actual Behavior:**
No error is shown, the picto animation for a file upload is seen undefinitely

## Environment

- **Open WebUI Version:** 0.3.2
- **Ollama (if applicable):** 0.1.42

- **Operating System:** Ubuntu 22
- **Browser (if applicable):** Chrome 125.0.6422.141

## Reproduction Details

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [X] I have included the browser console logs.
- [X] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
Request URL:  https://xxxxxx/rag/api/v1/doc
Request Method: POST
Status Code: 413 Request Entity Too Large
Remote Address: xxxx:443
Referrer Policy:  strict-origin-when-cross-origin

**Docker Container Logs:**

**Screenshots (if applicable):**
![OpenWebUi](https://github.com/open-webui/open-webui/assets/131153014/e45393c1-2d24-4ae5-bda8-e5a8fb9797d7)

## Installation Method

I'm using the docker installation for OpenWebUI

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tkg61** [comment]
i ran into this issue today and solved it via this doc: https://www.suse.com/support/kb/doc/?id=000021309 

Added this annotation: 
 nginx.ingress.kubernetes.io/proxy-body-size: "4096m"

**Comment by electronick-co** [comment]
Hi!

I had the same issue, followed this steps and problem solved!
https://stackoverflow.com/questions/24306335/413-request-entity-too-large-file-upload-issue
